.github/workflows/integration-tests.yml

```diff
@@ -23,7 +23,7 @@ jobs:
   # For now this is just MNE-Python, but others could be added
   mne:
     name: MNE-Python
-    runs-on: ubuntu-22.04
+    os: [ ubuntu-20.04, ubuntu-24.04, ubuntu-24.04-arm, ubuntu-22.04-arm ]
```

Comment:
I don't think we need to be testing the downstream packages on ARM, that would be for the downstream packages to do.